apps/prairielearn/src/lib/editors.ts

```diff
@@ -2082,3 +2200,45 @@ export class CourseInfoCreateEditor extends Editor {
     };
   }
 }
+
+export class MultiEditor extends Editor {
+  private editors: Editor[];
+
+  constructor(params: BaseEditorOptions, editors: Editor[]) {
+    super({
+      ...params,
+      description: editors.map((editor) => editor.description).join('; '),
```

Comment:
TODO: do some more clever things to produce a nicer description.

Reply:
That ended up being complicated. In ce3f39d I just forced users of `MultiEditor` to explicitly provide a description.